test.sh

```diff
@@ -25,7 +25,7 @@
 cd "$(dirname "$0")"
 
 lint_fail=0
-pylint *.py || {
+python3 -m pylint *.py || {
```

Comment:
I'd rather write pylint3 but this appears not to work in the docker image because of weird things.

Reply:
can you elaborate on the 'weird things' ?  Or leave a comment?